Documentation/dev/libvirt-howto.md

```diff
@@ -76,15 +77,23 @@ libvirtd_opts="--listen"
 
 Next, restart libvirt: `systemctl restart libvirtd`
 
-Finally, if you have a firewall, you may have to allow connections from the IP
-range used by your cluster nodes.  If you're using the default subnet of
-`192.168.124.0/24`, something along these lines should work:
+#### Firewall
+Finally, if you have a firewall, you may have to allow connections to the
+libvirt daemon from the IP range used by your cluster nodes.
+
+#### Manual management
+The following rule works for the suggested cluster ipRange of `192.168.124.0/24` and a libvirt *default* subnet of `192.168.122.0/24`:
 
 ```
-iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.124.1 --dport 16509 \
+iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.122.1 --dport 16509 \
   -j ACCEPT -m comment --comment "Allow insecure libvirt clients"
 ```
 
+If your uncertain about the libvirt *default* subnet you should be able to see its address using the command `
+ip -4 a show dev virbr0`.
```

Comment:
Another command to note would be `$ virsh --connect qemu:///system net-dumpxml default` to see how the default network has been configured